providers/obs/obs.go

```diff
@@ -248,16 +248,16 @@ func (b *Bucket) Iter(ctx context.Context, dir string, f func(string) error, opt
 	for {
 		output, err := b.client.ListObjects(input)
 		if err != nil {
-			return errors.Wrap(err, "fail to list object")
+			return errors.Wrap(err, "failed to list object")
 		}
 		for _, content := range output.Contents {
 			if err := f(content.Key); err != nil {
-				return errors.Wrap(err, "fail to call f for object")
+				return errors.Wrap(err, "failed to call iter function for object X")
 			}
 		}
 		for _, topDir := range output.CommonPrefixes {
 			if err := f(topDir); err != nil {
-				return errors.Wrap(err, "fail to call f for top dir object")
+				return errors.Wrap(err, "failed to call f for top dir object")
```

Comment:
Can we adjust this the same way as my previous comment (don't say `failed to call f ...` but rather `failed to call iter function`)